package.json

```diff
@@ -102,13 +101,13 @@
     "normalize.css": "^8.0.0",
     "prettier": "^1.15.3",
     "puppeteer": "^1.11.0",
-    "react": "^16.0.0",
+    "react": "^16.8.2",
     "react-ace": "^5.1.2",
     "react-codesandboxer": "^3.1.1",
     "react-custom-scrollbars": "^4.2.1",
     "react-docgen-typescript": "^1.12.2",
     "react-document-title": "^2.0.3",
-    "react-dom": "^16.0.0",
+    "react-dom": "^16.8.2",
```

Comment:
We will use this version only in our docs, we don't increase requirements there

Reply:
actually, would rather like to reserve the same React version being used both for Stardust, as well as in docs. This would ensure that if some consumer uses React version that satisfies Stardust requirements (say, React v16.4), then her environment would be enough to run all the examples and prototypes that are provided in docs - otherwise this is just a matter of time when Hooks will be used in an example's or prototype's code, and those won't be compatible with client's environment, in contrast to Stardust itself